alibi_detect/cd/utils.py

```diff
@@ -39,3 +43,67 @@ def update_reference(X_ref: np.ndarray,
             raise KeyError('Only `reservoir_sampling` and `last` are valid update options for X_ref.')
     else:
         return X_ref
+
+
+def activate_train_mode_for_dropout_layers(model: Union[nn.Module, tf.keras.Model], backend: str) -> Callable:
+    # TODO: Figure a way to do this properly for tensorflow models.
```

Comment:
Out of scope for current PR? If so, need to make sure to add an issue later.

Reply:
Yeah, no obvious way to do it. I'll add a note somewhere to be careful with using tf models and add an issue later.